src/panels/SkyBrowserPanel/WorldWideTelescope/Overlay.tsx

```diff
@@ -0,0 +1,37 @@
+import { Loader, Stack, Text } from '@mantine/core';
+
+import { useAppSelector } from '@/redux/hooks';
+
+import { useWwtProvider } from './WwtProvider/hooks';
+
+export function Overlay() {
+  const cameraInSolarSystem = useAppSelector(
+    (state) => state.skybrowser.cameraInSolarSystem
+  );
+  const { imageCollectionLoaded, wwtHasLoaded } = useWwtProvider();
+
+  if (!wwtHasLoaded || !imageCollectionLoaded) {
+    return (
+      <Stack align={'center'}>
+        {!wwtHasLoaded && <Text>Loading WorldWide Telescope...</Text>}
+        {wwtHasLoaded && !imageCollectionLoaded && (
+          <Text>Loading image collection...</Text>
+        )}
+        <Loader size={'lg'} type={'dots'} />
+      </Stack>
+    );
+  }
+
+  if (!cameraInSolarSystem) {
+    return (
+      <Stack align={'center'} p={'lg'}>
+        <Text>Camera is not in solar system.</Text>
+        <Text ta={'center'}>
+          You need to be in the solar system to use the SkyBrowser.
+        </Text>
```

Comment:
Do we want the upper text to be bolder or something? Feels weird that the bottom part is centered but the top isnt when they are the same size

Reply:
They are both centered, due to the centering of the stack
![image](https://github.com/user-attachments/assets/f24b00a0-1ee1-41e8-91c7-7ae1566fdd19)


However, for clarit I suggest to move the `ta={`'center'}` to the stack instead: 
```
      <Stack align={'center'} ta={'center'} p={'lg'}>
        <Text>Camera is not in solar system.</Text>
        <Text>You need to be in the solar system to use the SkyBrowser.</Text>
```